ivadomed/loader.py

```diff
@@ -160,18 +160,20 @@ def _check_isMRIparam(mri_param_type, mri_param):
         super().__init__(self.filename_pairs, slice_axis, cache,
                          transform, slice_filter_fn, canonical)
 
-    def filter_roi(self, nb_nonzero_thr):
-        filter_indexes = []
-        for segpair, slice_roi_pair in self.indexes:
-            roi_data = slice_roi_pair['gt']
-            if not np.any(roi_data):
-                continue
-            if np.count_nonzero(roi_data) <= nb_nonzero_thr:
-                continue
-
-            filter_indexes.append((segpair, slice_roi_pair))
-
-        self.indexes = filter_indexes
+
+def filter_roi(ds, nb_nonzero_thr):
```

Comment:
could you please add docstrings using to understand the scope of the function (and what i/o), thanks. Since we don't have a convention (see https://github.com/neuropoly/ivado-medical-imaging/issues/156), you can use the [PEP257 docstring convention](https://www.python.org/dev/peps/pep-0257/).

Reply:
Documented :-)